tests/testthat/setup.R

```diff
@@ -0,0 +1,11 @@
+old_opts = options(
+  warnPartialMatchArgs = TRUE,
+  warnPartialMatchAttr = TRUE,
+  warnPartialMatchDollar = TRUE
+)
+
+# https://github.com/HenrikBengtsson/Wishlist-for-R/issues/88
+old_opts = lapply(old_opts, function(x) if (is.null(x)) FALSE else x)
+
+old_plan = future::plan()
+future::plan("sequential")
```

Comment:
Is the future package necessary for mlr3cluster? I think you can remove these lines and also update "Suggests" in the DESCRIPTION.

Reply:
I copy/pasted this from mlr3 tests without thinking. I removed this now but I don't think future is necessary for mlr3clutser.